### PR TITLE
Reusable add-to-map button component

### LINKED ISSUE #4464: Unify naming of the same action ("add to map")
# Bug

**Describe the bug**
Currently, the wording of "Add to map" action is different based on which datasource the user is adding.

**Expected behavior**
The same action shall be named equally in all places of the app. Just pick one of the options and use it everywhere. Including/excluding the "+" icon.

**Screenshots**
## Data Catalogue
image
## URL
image
## File
image

### PR DESCRIPTION
**Pull request by FilipLeitner**
## Description

Unify naming of the same action ("add to map") by reusing the same component

## Related issues or pull requests

closes #4464 

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Dependency updates
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe)
- [ ] I am unsure (we'll look into it together)

## Do you introduce a breaking change?

- [ ] Yes
- [x] No
- [ ] I am unsure (no worries, we'll find out)

## Checklist

- [x] I understand and agree that the changes in this PR will be licensed under the [MIT License]
- [ ] I have followed the [guidelines for contributing](https://github.com/hslayers/hslayers-ng/blob/master/CONTRIBUTING.md)
- [ ] The proposed change fits to the content of the [code of conduct](https://github.com/hslayers/hslayers-ng/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have added or updated tests and documentation, and the test suite passes (run `npm test` locally)
- [ ] I'm lost; why do I have to check so many boxes? Please help!

### CODE REVIEW

File: projects/hslayers/src/common/add-to-map/add-to-map.component.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      }
+    `,
+  ],
+  imports: [CommonModule, HsLanguageModule],
+})
+/**
+ * title: Translation string to be used title
+ */
+// eslint-disable-next-line @angular-eslint/component-class-suffix
+export class HsAddToMapButton {
```
Comment (jmacura) [inline projects/hslayers/src/common/add-to-map/add-to-map.component.ts:45]: Any reason why not to call this class 
```suggestion
export class HsAddToMapButtonComponent {
```
?

File: projects/hslayers/src/common/add-to-map/add-to-map.component.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  @Input() disabled: boolean;
+  @Input() title: string;
```
Comment (jmacura) [inline projects/hslayers/src/common/add-to-map/add-to-map.component.ts:47]: It creates floating "undefined" in Firefox when not specified in host component. Can we place 'COMMON.addToMap' also here as default?
obrazek

**Review by jmacura — COMMENTED** [review]
Nice solution. See my comments regarding some details.

File: projects/hslayers/src/common/add-to-map/add-to-map.component.ts (hunk 2 above)
Comment (FilipLeitner) [reply projects/hslayers/src/common/add-to-map/add-to-map.component.ts]: [2fbf277](https://github.com/hslayers/hslayers-ng/pull/4468/commits/2fbf27703cb93a15bb3604b2d187c615a9374370)

File: projects/hslayers/src/common/add-to-map/add-to-map.component.ts (hunk 1 above)
Comment (FilipLeitner) [reply projects/hslayers/src/common/add-to-map/add-to-map.component.ts]: [2fbf277](https://github.com/hslayers/hslayers-ng/pull/4468/commits/2fbf27703cb93a15bb3604b2d187c615a9374370)

**Review by jmacura — APPROVED** [review]

**Pull request merged**